### DISCUSSION TITLE
For what is the player history?

### DISCUSSION THREAD

**Discussion by BlyZeDev** [lavalink4net/Lavalink4NET#137] [Q&A] [answered]
```cs
QueuedLavalinkPlayer.Queue //I get for what that is
QueuedLavalinkPlayer.Queue.History //But what does this exactly?
```

**Comment by angelobreuer** [comment, marked as answer]
Hi @BlyZeYT ,

the player history is a collection of tracks that the player has finished playing. For example, if you have two tracks in a queue and the player is currently playing a track, after the track ends, the history will contain the track that the player was playing and the queue will pop one item for the next track. You can use the history to look up what the player finished playing.
